yarn.lock

```diff
@@ -2309,10 +2318,16 @@ aws4@^1.8.0:
   resolved "https://registry.yarnpkg.com/aws4/-/aws4-1.8.0.tgz#f0e003d9ca9e7f59c7a508945d7b2ef9a04a542f"
   integrity sha512-ReZxvNHIOv88FlT7rxcXIIC0fPt4KZqZbOlivyWtXLt8ESx84zd3kMC6iK5jVeS2qt+g7ftS7ye4fi06X5rtRQ==
 
-babel-compat@0.21.4:
-  version "0.21.4"
-  resolved "https://registry.npmjs.org/babel-compat/-/babel-compat-0.21.4.tgz#f815a8adc5bad2192f2288090feab37288afbf30"
-  integrity sha512-SJ6MiMKhKD/mCBQgD4u6cz8X/dRPdL6pvMu+pQoDOIX9tbFHcv6FI3dZ06KG0ywx9w8L6L5RtLoc+/qXclZbcg==
+babel-compat@0.21.5:
+  version "0.21.5"
+  resolved "https://registry.yarnpkg.com/babel-compat/-/babel-compat-0.21.5.tgz#477afdb7a61515b50a81e90cc29171f37a7a54dc"
```

Comment:
Is that ok to switch from https://registry.npmjs.org to https://registry.yarnpkg.com?